packages/react-core/src/components/Nav/NavItem.tsx

```diff
@@ -31,7 +31,7 @@ export interface NavItemProps extends Omit<React.HTMLProps<HTMLAnchorElement>, '
   /** Component used to render NavItems if  React.isValidElement(children) is false */
   component?: React.ElementType<any> | React.ComponentType<any>;
   /** Flyout of a nav item. This should be a Menu component. Should not be used if the to prop is defined. */
-  flyout?: React.ReactElement;
+  flyout?: React.ReactElement<any>;
```

Comment:
seems like we're introducing a lot of `<any>` with this change... wouldn't something like `React.ReactNode` be a better choice in this and several other cases? it's essentially doing no type checking.

Reply:
Per discussion during sprint planning, these updates that were made towards 11368 we are going to mark as expected/complete in terms of running the codemod only. Followup work to update these types should be handled in a followup as part of breaking changes.

Reply:
sounds good, as long as there's a follow-up to address, i can approve - thanks! 👍